mmpose/models/detectors/top_down.py

```diff
@@ -153,26 +154,27 @@ def forward_train(self, img, target, target_weight, img_metas, **kwargs):
             # target_weight: [batch_size, num_joints, 1]
             losses['mse_loss'] = self.loss(output, target, target_weight)
 
-        if isinstance(output, list):
-            if target.dim() == 5 and target_weight.dim() == 4:
-                _, avg_acc, _ = pose_pck_accuracy(
-                    output[-1].detach().cpu().numpy(),
-                    target[:, -1, ...].detach().cpu().numpy(),
-                    target_weight[:, -1,
-                                  ...].detach().cpu().numpy().squeeze(-1) > 0)
-                # Only use the last output for prediction
+        if self.target_type == 'GaussianHeatMap':
+            if isinstance(output, list):
+                if target.dim() == 5 and target_weight.dim() == 4:
+                    _, avg_acc, _ = pose_pck_accuracy(
+                        output[-1].detach().cpu().numpy(),
+                        target[:, -1, ...].detach().cpu().numpy(),
+                        target_weight[:, -1,
+                                      ...].detach().cpu().numpy().squeeze(-1) >
+                        0)
+                    # Only use the last output for prediction
```

Comment:
where does this line of comment applies to, the if part, or the else part?

Reply:
this comment is inherit from the origin mmpose project

Reply:
I see, it should be placed before L163